modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/BuilderScreen/BuilderListItem.tsx

```diff
@@ -28,10 +29,10 @@ interface Iprops {
 	isDefaultSort?: boolean;
 	label?: string;
 	objectFieldName: string;
+	onEditing?: (boolean: boolean) => void;
 	onEditingObjectFieldName?: (objectFieldName: string) => void;
-	onEditingSort?: (boolean: boolean) => void;
-	onVisibleModal?: (boolean: boolean) => void;
-	sortOrder?: string;
+	onVisibleEditModal?: (boolean: boolean) => void;
+	secondColumntext?: string;
```

Comment:
Hi @danvb112, just a suggestion, I think this name `secondColumnText` and also `secondColumnHeader` maybe could be renamed by something more clear. If I'm not wrong, it's related  do sorting, isn't it? If so, you could use `sortingColumnText` and `sortingColumnHeader`, or something like that.

Reply:
Done, Thanks @alinedoleron! :smile: